### DISCUSSION TITLE
How to get the equivalent rebasing/evolving to `hg sync -a`?

### DISCUSSION THREAD

**Discussion by TomerAberbach** [jj-vcs/jj#4520] [Q&A] [answered]
I recently left Google, started missing the Fig experience, and was delighted to find `jj`!

One thing I'm trying to figure out is how to get the equivalent behavior to `hg sync -a` when fetching from the Git remote.

For example, let's say I have some change that I created a pull request for, and then I merged that pull request. If I do `jj git fetch`, then I get that new information, but none of the changes are automatically evolved (unlike Fig did when I was using it).

I seem to be able to run:
```sh
$ jj rebase --skip-emptied -d main
```

To get an changes that were on top of the merged PR rebased/evolved, but that doesn't seem to affect changes that were not related to the PR, but are now still on top of the old `HEAD`. To fix that I seem to have to manually rebase each of these changes onto the new `HEAD`.

Is there a sequence of commands I can use to replicate the `hg sync -a` behavior?

**Comment by PhilipMetzger** [comment, marked as answer]
We're still missing a `sync` command, see #1039. In the meantime you'll have to rebase and abandon the commits manually.

**Reply by TomerAberbach** [reply]
Ah got it, thanks!
